### PR TITLE
Members only section 

### PR DESCRIPTION
**Pull request by tornglintaffychen**
able to see the orders in My Account

### CODE REVIEW

File: browser/js/common/directives/navbar/navbar.html [hunk 1]
```diff
@@ -5,14 +5,14 @@
         </div>
 
         <ul class="nav navbar-nav">
-            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn()">
+            <li ng-repeat="item in items" ng-show="!item.auth || isLoggedIn() && user.firstName !== 'Unregistered'">
```
Comment (kbuechner) [inline browser/js/common/directives/navbar/navbar.html:8]: will this ever be true if we're using bella?

File: server/app/routes/users/index.js [hunk 2]
```diff
@@ -44,10 +45,10 @@ router.put('/:id', function (req, res, next) {
 });
 //combined*sv
 //get one user, their order and reviews
-router.get('/:id', function (req, res, next) {
+router.get('/member', function (req, res, next) {
```
Comment (kbuechner) [inline server/app/routes/users/index.js:48]: are we doing this for protection? i.e. are we obscuring the member id for security reasons?

File: browser/js/common/directives/navbar/navbar.html (hunk 1 above)
Comment (tornglintaffychen) [inline browser/js/common/directives/navbar/navbar.html:8]: it's Unregistered now

File: server/app/routes/users/index.js (hunk 2 above)
Comment (tornglintaffychen) [inline server/app/routes/users/index.js:48]: security is not the feature on this branch

**Pull request merged**